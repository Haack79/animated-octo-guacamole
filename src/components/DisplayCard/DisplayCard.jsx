import React from 'react'; 

export const DisplayCard = () => {
    return (
        <>
            <section>
                <h1>Title</h1>
                <video></video>
                <p>text here</p>
                <p className="comments">comments here</p>
            </section>
        </>
    )
}