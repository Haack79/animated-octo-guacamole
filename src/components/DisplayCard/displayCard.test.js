import { shallow } from 'enzyme'; 
import {DisplayCard} from './DisplayCard';
import React from 'react';
import {render, screen} from '@testing-library/react';
describe('DisplayCard', () => {
    test('renders DisplayCard', () => {
        render(<DisplayCard />);
        screen.debug();
    });
});
it('to render displayCard component', () => {
    expect(shallow(<DisplayCard />).length).toEqual(1); 
});

it('expect to render a snapshot of component', () => {
    expect(shallow(<DisplayCard />)).toMatchSnapshot();
})
