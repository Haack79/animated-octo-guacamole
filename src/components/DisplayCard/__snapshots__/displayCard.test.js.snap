// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render a snapshot of component 1`] = `ShallowWrapper {}`;
